CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.40.0 - Unreleased
+
+- The previously deprecated `update` function of the `dict` has been replaced
```

Comment:
I am not fond of this changelog entry.

Reply:
I think we can just say "The `dict` module gains the `update` function.".